chapter_introduction/index_vn.md

```diff
@@ -2559,7 +2569,13 @@ With what we know today, this strikes us a much more pressing concern
 than the potential of malevolent superintelligence to destroy humanity.
 -->
 
-*dịch đoạn phía trên*
+Một sự lo lắng khác là Ai có thể sử dụng trong cuộc sống thường nhật như thế nào.
+Nhiều khả năng rất nhiều tác vụ đơn giản đang được thực hiện bởi tài xế xe tải và trợ lý cửa hàng có thể và sẽ bị tự động hóa.
+Các robot nông trại sẽ nhiều khả năng giảm giá của nông nghiệp hữu cơ mà còn tự động quá trình thu hoạch.
```

Comment:
```suggestion
Các robot nông trại sẽ nhiều khả năng làm giảm chi phí của nông nghiệp hữu cơ nhưng cũng sẽ tự động hóa quá trình thu hoạch.
```

Reply:
Anh nghĩ đây là cấu trúc "not only... but also" nên sẽ dịch là "không những ... mà còn".